beetsplug/web/__init__.py

```diff
@@ -42,6 +42,10 @@ def _rep(obj, expand=False):
         else:
             del out['path']
 
+        # Filter all bytes attributes and convert them to strings
+        for key in filter(lambda key: isinstance(out[key], bytes), out):
```

Comment:
A somewhat more "idiomatic Python" way of doing this would be to use a plain `if`. For example:

```py
for key, value in out.items():
    if isinstance(value, bytes):
        out[key] = ...
```